app/views/layouts/_analytics.html.erb

```diff
@@ -14,4 +14,4 @@
     ga('set', 'forceSSL', true);
     ga('send', 'pageview');
   </script>
-<% end %>
+<% end %>
```

Comment:
hey this means there's no newline at EOF!  are you perhaps using sublime? http://docs.sublimetext.info/en/latest/reference/settings.html